c_glib/arrow-glib/array-builder.cpp

```diff
@@ -231,8 +231,14 @@ garrow_array_builder_append_values(GArrowArrayBuilder *builder,
   if (n_remains > 0) {
     ++n_loops;
   }
+#ifdef _MSVC_LANG
+  std::vector<uint8_t> data_vec(value_size * chunk_size);
+  uint8_t *data = data_vec.data();
```

Comment:
We can always use this style. (We can remove `uint8_t data[value_size * chunk_size]` style.)

Reply:
My understanding is that the variable length array approach would use stack allocation so there might be a performance benefit over using `std::vector`, but I'm happy to only use `std::vector` if you're not worried about that.

Reply:
Right. I want to choose easy to maintain for now. But we may rethink about this later.